### DISCUSSION TITLE
Not working when passing multiple inputs in PromptTemplate

### DISCUSSION THREAD

**Discussion by vanshsehgal2001** [langflow-ai/langflow#185] [Q&A]
**Describe the bug**
I was trying to replicate the functionality in langchain in langflow. I tried using Prompt Template. Its working fine for single input but its not throwing error when multiple inputs are passed.

**Browser and Version**
 - Browser chrome
 - Version 112.0.5615.137 (Official Build) (x86_64)

**To Reproduce**
I simply installed langflow and ran it on local. Attaching screenshots for reference.

**Screenshots**
Below are the screenshots of single input and its working fine.

<img width="1440" alt="Screenshot 2023-04-24 at 10 52 04 AM" src="https://user-images.githubusercontent.com/125663043/233906854-84b73391-4f8f-4b81-83a8-40ac4179ad4f.png">
<img width="1440" alt="Screenshot 2023-04-24 at 10 53 06 AM" src="https://user-images.githubusercontent.com/125663043/233907102-f92e3d05-1f61-40fe-8a70-6e2fddf9cd4e.png">

Below are the screenshots when passing multiple inputs and error is thrown.

<img width="1440" alt="Screenshot 2023-04-24 at 10 51 26 AM" src="https://user-images.githubusercontent.com/125663043/233907672-fa47f27c-eab1-4f34-8ac8-8d9fef2acbf5.png">
<img width="1440" alt="Screenshot 2023-04-24 at 10 51 34 AM" src="https://user-images.githubusercontent.com/125663043/233907898-cac20589-0458-4c41-b6bd-bb060783cf77.png">
<img width="1440" alt="Screenshot 2023-04-24 at 10 51 51 AM" src="https://user-images.githubusercontent.com/125663043/233908069-e90e20f7-bb86-4aaa-a428-48cc2ad1bf89.png">

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
